kmk/extensions/media_keys.py

```diff
@@ -0,0 +1,28 @@
+from kmk.extensions import Extension
+from kmk.keys import make_consumer_key
+
+
+class Layers(Extension):
```

Comment:
Is there a particular reason this is named the same as the other `Layers` extension?

Reply:
CopyPaste error. good catch.